streamly-process.cabal

```diff
@@ -1,96 +1,103 @@
 cabal-version:       2.2
 name:                streamly-process
 version:             0.1.0.0
-synopsis:            Stream Interfaces For System Processes
+synopsis:            Streaming interfaces for system processes
 description:
-  Streamly is a framework for writing programs in a high level, declarative
-  data flow programming paradigm. It provides a simple API, very close to
-  standard Haskell lists.
-  .
-  This package allows us to extract streams from Processes
-  .
-homepage:            https://github.com/composewell/streamly-process
+  Connect system processes with Haskell functions or with each other
+  like Unix pipes using streamly streams.
+homepage:            https://streamly.composewell.com
 bug-reports:         https://github.com/composewell/streamly-process/issues
 license:             Apache-2.0
 license-file:        LICENSE
 author:              Composewell Technologies
 maintainer:          streamly@composewell.com
 copyright:           Composewell Technologies
 category:            Control, Concurrency, Streaming, Processes
+stability:           Experimental
 build-type:          Simple
 extra-source-files:
-    .github/workflows/haskell.yml
-    .gitignore
-    .hlint.yaml
-    appveyor.yml
-    stack.yaml
-    CHANGELOG.md
-    README.md
-    NOTICE
-    cabal.project.ci
-    design/proposal.md
+  CHANGELOG.md
+  NOTICE
+  README.md
+  design/proposal.md
+source-repository head
+  type: git
+  location: https://github.com/composewell/streamly-process
 
-library
+common compile-options
+  default-language: Haskell2010
+  ghc-options:
+    -Wall
+    -Wcompat
+    -Wunrecognised-warning-flags
+    -Widentities
+    -Wincomplete-record-updates
+    -Wincomplete-uni-patterns
+    -Wredundant-constraints
+    -Wnoncanonical-monad-instances
+
+common optimization-options
   ghc-options:
-                -Wall
-                -Wcompat
-                -Wunrecognised-warning-flags
-                -Widentities
-                -Wincomplete-record-updates
-                -Wincomplete-uni-patterns
-                -Wredundant-constraints
-                -Wnoncanonical-monad-instances
+    -O2
+    -fdicts-strict
+    -fspec-constr-recursive=16
+    -fmax-worker-args=16
 
+library
+  import: compile-options, optimization-options
   hs-source-dirs: src
+  exposed-modules:
+    Streamly.System.Process
+  ghc-options:
+    -Wall
+    -Wcompat
+    -Wunrecognised-warning-flags
+    -Widentities
+    -Wincomplete-record-updates
+    -Wincomplete-uni-patterns
+    -Wredundant-constraints
+    -Wnoncanonical-monad-instances
   build-depends:
-      streamly          >= 0.7.1 && < 0.8
-    , primitive         >= 0.5.4 && < 0.8
-    , base              >= 4.8   && < 5
+      base              >= 4.8   && < 5
     , process           >= 1.0   && < 1.7
-    , transformers      >= 0.4 && < 0.6
+    , streamly          >= 0.7.1 && < 0.8
```

Comment:
Can you open an issue to update streamly and assign it to me? Or are you doing it?

Reply:
I have updated it in the next change.